aten/src/ATen/MemoryFormatUtils.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <ATen/ATen.h>
+
+namespace at {
+
+static Tensor clone_if_possible_with_memory_format(const Tensor& self) {
```

Comment:
`static` functions in header files don't make sense I think. Usually one marks a function as `static` so that it is only used in the file it is defined.